lib/peerconnection.js

```diff
@@ -204,8 +204,8 @@ RTCPeerConnection.prototype.addTransceiver = function addTransceiver() {
   return this._pc.addTransceiver.apply(this._pc, arguments);
 };
 
-RTCPeerConnection.prototype.addTrack = function addTrack(track, stream) {
-  return this._pc.addTrack(track, stream);
+RTCPeerConnection.prototype.addTrack = function addTrack(track, streams) {
+  return this._pc.addTrack(track, streams);
```

Comment:
Please check out the Web IDL for `addTrack` [here](https://w3c.github.io/webrtc-pc/#rtcpeerconnection-interface-extensions) — `streams` should use the [rest parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) syntax. So, at least in the JavaScript layer here, we should define this method as

```js
RTCPeerConnection.prototype.addTrack = function addTrack(track, ...streams) {
  return this._pc.addTrack(track, streams);
};
```

node-webrtc supports Node 8 onwards, so I believe we ca safely use rest parameter syntax here.

Reply:
Will do.